internal/oci/spec/spec.go

```diff
@@ -110,6 +110,12 @@ func New(o ...Option) (*runtime.Spec, error) {
 				Source:      "tmpfs",
 				Options:     []string{"nosuid", "strictatime", "mode=755", "size=65536k"},
 			},
+			{
+				Type:        "tmpfs",
+				Destination: "/xfntmp",
```

Comment:
Would it make sense to use plain-old `/tmp`?

Reply:
I wasn't sure if it's possible to override the existing `/tmp` folder's permissions from this config. And looks like, we can do it. Updated accordingly.